test/utils.js

```diff
@@ -0,0 +1,30 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const ParserError = require('../lib/errors/parser-error');
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+
+
```

Comment:
please remove this extra line, I suggest running locally `npm run lint` to see all the linter errors